app/serializers/answer_serializer.rb

```diff
@@ -1,13 +1,3 @@
-class AnswerSerializer < ActiveModel::Serializer
-  attributes :id, :user, :content, :question_id, :votes_count, :created_at, :updated_at, :comments_count
-
-  def user
-    {
-      id: object.user.id,
-      name: object.user.name,
-      email: object.user.points,
-      image: object.user.social_providers.first.try(:profile_picture),
-      points: object.user.points
-    }
-  end
+class AnswerSerializer < MainSerializer
+  attributes :question_id, :created_at, :updated_at, :comments_count
```

Comment:
`:created_at` and `:updated_at` are already in *MainSerializer*, I don't think there is a need for them here.